### DISCUSSION TITLE
How to share two folders between host and guest

### DISCUSSION THREAD

**Discussion by bjnobre** [virtio-win/kvm-guest-drivers-windows#1108] [Q&A]
I'm guesting w10 and trying to access two different folders, but it doesn't work. If I set it on VM, I get mapped on guest the same folder twice!

Is it possible to do what I'm trying to?

**Comment by YanVugenfirer** [comment]
Hello @bjnobre,

Did you try to follow the "Multiple virtiofs instances" section in https://virtio-win.github.io/Knowledge-Base/Virtiofs:-Shared-file-system#multiple-virtiofs-instances?

Best regards,
Yan.

**Reply by SampsonF** [reply]
I am able to use, manually:
launchctl-x64.exe start virtiofs viofsW mount_tag0 w:
launchctl-x64.exe start virtiofs viofsX mount_tag1 x:

to mount 2 drive letters to 2 virtiofs folders.

Just want to know how to auto mount w: and x: when Windows booted up and before any user log in.
